sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnCastUtil.scala

```diff
@@ -90,6 +90,26 @@ object ColumnCastUtil extends Arm {
           } else {
             cv
           }
+        case m: MapType =>
+          // map is list of structure
+          val struct = cv.getChildColumnView(0)
+          toClose += struct
+          
+          if(cv.getType != DType.LIST || struct.getType != DType.STRUCT) {
+            throw new IllegalStateException("Map should be List(Structure) in column view")
+          }
+
+          val newChild = convertTypeAToTypeB(struct,
+            StructType(Array(StructField("", m.keyType), StructField("", m.valueType)))
+            , predicate, toClose)
```

Comment:
Nit:  Normally the comma is placed on the same line as the parameter before it?  It's not consistent even within this method call.

Reply:
done